nextcord/embeds.py

```diff
@@ -374,7 +399,7 @@ def image(self) -> _EmbedMediaProxy:
         """
         return EmbedProxy(getattr(self, "_image", {}))  # type: ignore
 
-    def set_image(self, url: Optional[Any]) -> Self:
+    def set_image(self, url: Optional[Any] = None, *, file: Optional[File] = None) -> Self:
```

Comment:
It would be nice if the `url` parameter was changed to accept both urls and files. It would be ideal (but a breaking change) if both the url and file parameters were keyword only.

Reply:
so i'll apply this to all the embeds methods that supports File(s)